src/riak_kv.proto

```diff
@@ -406,11 +403,7 @@ message TsRow {
 message TsCell {
   optional bytes binary_value = 1;
   optional sint64 integer_value = 2;
```

Comment:
This WAS discussed today during and after the standup, and there is much less of a standard around what to call different integers of different lengths in different languages.